### DISCUSSION TITLE
LagLife is your username on youtube?

### DISCUSSION THREAD

**Discussion by Dhyfer1** [bbmaster123/10SM#26] [General]
I was just looking at the readme and I found this:

>See 10SM in action: LagLife — Reverting Windows 11 UI (Pt.2)
Be sure to check out part 1 and 3 for other interesting tweaks and windows shenanigans!

So are you or are you not the LagLife user? I need to contact him, but I don't know how.

**Comment by bbmaster123** [comment]
Sorry no, I am not LagLife
Have you tried leaving a comment on his yt channel? That's how I originally got in touch

**Reply by Dhyfer1** [reply]
Ok. Yes, I have left some comments on several of his videos, but I am still waiting for his response, and there is no link on his channel to contact him by other means.
